codewit/Dockerfile.backend

```diff
@@ -0,0 +1,13 @@
+FROM node:20-alpine
+
+WORKDIR /codewit
+
+COPY package* .
+
+RUN npm install && npm i -g nx
+
+EXPOSE 3000
+
+COPY . .
+
+CMD nx run api:serve:development
```

Comment:
These Dockerfiles are helpful, but please also update the `README.md` to have a subsection under **Getting Started** with quick instructions on using Docker Compose with the project.